File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/ElongatedBone.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class ElongatedBone extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_elongated_bone_name;
        this.idDescription = R.string.item_elongated_bone_description;
        this.idImage = R.drawable.elongated_bone;
        this.source.add(Integer.valueOf(R.string.dungeon_name_eternal_battlefield));
        this.source.add(Integer.valueOf(R.string.raid_name_ancient_grave_digging));
        this.price = 6L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/ElongatedBone.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class ElongatedBone : Item() {
    override fun configureProperties() {
        idName = R.string.item_elongated_bone_name
        idDescription = R.string.item_elongated_bone_description
        idImage = R.drawable.elongated_bone
        source.add(R.string.dungeon_name_eternal_battlefield)
        source.add(R.string.raid_name_ancient_grave_digging)
        price = 6L
    }
}
